src/routes/safe/components/Transactions/TxList/MethodDetails.tsx

```diff
@@ -24,7 +25,7 @@ export const MethodDetails = ({ data }: { data: DataDecoded }): React.ReactEleme
 
       {data.parameters?.map((param, index) => {
         const isArrayValueParam = isArrayParameter(param.type) || Array.isArray(param.value)
-        const inlineType = !isArrayValueParam && isAddress(param.type) ? 'address' : undefined
+        const inlineType = isAddress(param.type) ? 'address' : isBytes(param.type) ? 'bytes' : undefined
```

Comment:
Might be worth moving this to a helper function. Is it only used here?

Reply:
Only used here. In the other places that we call `<TxDataRow />` we pass the `inlineType` non-programatically.